modules/common/src/main/scala/emil/builder/package.scala

```diff
@@ -160,6 +160,25 @@ package builder {
 
     def withLength(len: F[Long]): Attach[F] =
       copy(attach = attach.copy(length = len))
+
+    def withDisposition(disp: Disposition): Attach[F] =
+      copy(attach = attach.copy(disposition = Some(disp)))
+
+    def withDisposition(disp: Option[Disposition]): Attach[F] =
+      copy(attach = attach.copy(disposition = disp))
+
+    def withContentId(cid: String): Attach[F] =
+      copy(attach = attach.copy(contentId = Some(cid)))
+
+    def withContentId(cid: Option[String]): Attach[F] =
+      copy(attach = attach.copy(contentId = cid))
+
+    def withInlinedContentId(cid: Option[String]): Attach[F] =
```

Comment:
`def withInlinedContentId(cid: Option[String]): Attach[F]`

^- after my own review, I think this utility method makes no sense in the API.

Reply:
I agree, it is good enough to provide only the "single" modifiers. Users can easily create their own things based on that.